modules/custom/az_event/config/install/core.entity_view_display.node.az_event.az_row_with_background.yml

```
@@ -0,0 +1,286 @@
+        id: ''
+        element: div
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+    group_column_text:
+      children:
+        - smart_title
+        - group_when
+      label: 'Column - Text'
+      parent_name: group_row
+      region: content
+      weight: 10
+      format_type: html_element
+      format_settings:
+        classes: col-9
+        show_empty_fields: false
+        id: ''
+        element: div
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+    group_month:
+      children:
+        - az_event_month
+      label: Month
+      parent_name: group_background_date
+      region: content
+      weight: 3
+      format_type: html_element
+      format_settings:
+        classes: 'h5 text-center mt-0 mb-0 text-uppercase hover-text-underline'
+        show_empty_fields: false
+        id: ''
+        element: div
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+    group_day:
+      children:
+        - az_event_day
+      label: Day
+      parent_name: group_background_date
+      region: content
+      weight: 4
+      format_type: html_element
+      format_settings:
+        classes: 'h4 text-center mt-0 mb-0 hover-text-underline'
+        show_empty_fields: false
+        id: ''
+        element: div
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+    group_where:
+      children:
+        - field_az_location
+      label: Where
+      parent_name: ''
+      region: hidden
+      weight: 15
+      format_type: html_element
+      format_settings:
+        classes: 'nolink mt-3'
+        show_empty_fields: false
+        id: ''
+        element: div
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+    group_when:
+      children:
+        - field_az_event_date
+      label: When
+      parent_name: group_column_text
+      region: content
+      weight: 2
+      format_type: html_element
+      format_settings:
+        classes: 'text-muted font-weight-normal'
+        show_empty_fields: false
+        id: ''
+        element: div
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+    group_background_date:
+      children:
+        - group_month
+        - group_day
+      label: 'Background - Date'
+      parent_name: group_column_date
+      region: content
+      weight: 14
+      format_type: html_element
+      format_settings:
+        classes: 'bg-azurite h-100 py-3'
```

Don't extend the blue box around the date to the full height of the row (same behavior as on the Calendar page):
```suggestion
        classes: 'bg-azurite py-3'
```